composer.json

```diff
@@ -18,6 +18,7 @@
     },
     "require": {
         "vlucas/phpdotenv": "^2.2",
-        "consolidation/robo": "^1.1.0"
+        "consolidation/robo": "^1.3",
+        "drush/drush": "9.4.0"
```

Comment:
Unless there's a reason to lock this exact version, we should use semvar here instead:
```json
"drush/drush": "^9.4"
```

Reply:
@wxactly sadly, there is https://github.com/thinkshout/robo-drupal/issues/18#issuecomment-436047051

Reply:
Haha lovely... just lovely. Welp, had to check 🤷‍♂️